mas-foundation/build.gradle

```diff
@@ -206,6 +153,85 @@ artifacts {
     archives sourcesJar
 }
 
+afterEvaluate {
```

Comment:
I assume, we should do the same for other modules in this repo. Don't we?

Reply:
yes.. we are going to do the same for other repos.